python/vast/utils/arrow.py

```diff
@@ -35,12 +35,17 @@ def __arrow_ext_scalar_class__(self):
         return PatternScalar
 
 
-class AddressScalar(pa.ExtensionScalar):
+class IPScalar(pa.ExtensionScalar):
     def as_py(self) -> ip.IPv4Address | ip.IPv6Address | None:
         return None if self.value is None else unpack_ip(self.value.as_py())
 
 
-class AddressType(pa.ExtensionType):
+class IPType(pa.ExtensionType):
+    # NOTE: The identifier for the extension type of VAST's ip type has not
+    # changed when the type was renamed from address to ip because that would be
+    # a breaking change. This is fixable by registering two separate extension
+    # types with the same functionality but different ids, but that's a lot of
+    # effort for something users don't usually see.
```

Comment:
(Anchor comment for the existing discussion on this topic.)

Reply:
As I said above, let's consider doing this in a follow-up story.